examples/image_processing/reslice.py

```diff
@@ -0,0 +1,81 @@
+"""
+==========================================================
+Load CT slices and plot axial, sagittal and coronal images
+==========================================================
+
+.. usage:
+
+   reslice.py <glob>
```

Comment:
This now shows up in the tooltip. Maybe add a description above the usage better suited for the tooltip (see other examples)

Reply:
Nonetheless - really nice example :)

Reply:
Hey guys, a little question about loading of the files - I can't quite grasp the meaning of the 'glob' term. If I have a collection of folders with .dcm files, how can I reslice them? I started by loading the .dcm files and sorting them:
TestPath = "/home/blue/Desktop/TEST/"
# Store all dicom files in a list
Dicom = []
# Check the directory for .dcm files and add them to the list
for dirName, subdirList, fileList in os.walk(TestPath):
    for filename in fileList:
        if ".dcm" in filename.lower():
            Dicom.append(os.path.join(dirName,filename))

# Sort the Slices in the list
Dicom = sorted(Dicom, key=lambda i: int(os.path.splitext(os.path.basename(i))[0]))

if I have a sorted list of .dcm files, what should I do next?

Reply:
A glob is something like '*.dcm' (if your files all have the .dcm extension). If you have DICOM files in different directories, are they different 3D scans?

Reply:
Yeah, I have several dcm files, each one of them is a different 3d scan.